### PR TITLE
udemy phase#1

### PR DESCRIPTION
**Pull request by shereen-fathy**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+                <span class="title">Courses on sale from E£169.99</span>
+                <span>
+                  | Thanks for enrolling in a free course. Now, upgrade your
+                  learning.
+                </span>
+              </span>
+              <span class="un-title"> Ends in 5h 52m 34s.</span>
+            </div>
+          </div>
+          <button class="remove-text">x</button>
```
Comment (shereen-fathy) [inline index.html]: I think this should be `btn-remove` or `btn-close`, right?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+              <a href="/" class="">
+                <i class="fa-solid fa-cart-shopping"></i>
+              </a>
+            </li>
+
+            <li><a href="/" class="login"> Login </a></li>
+            <li><a href="/" class="signup"> Signup </a></li>
+            <li>
+              <a href="/" class="global-icon"
+                ><i class="fa-solid fa-globe"></i>
```
Comment (shereen-fathy) [inline index.html:64]: `global-icon` shouldn’t be a link, right?

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,448 @@
...
+}
+.btn-explore a {
+  padding: 1rem;
+  border: 1px solid #000;
+  color: #000;
+  font-weight: 800;
+}
+/* 
+
+@media (max-width: 885px) {
```
Comment (shereen-fathy) [inline styles.css]: please remove unneeded comments

**Comment by shereen-fathy** [comment]
@waelelsafty07 left minor comments, otherwise looks great, Thank you!

File: index.html (hunk 1 above)
Comment (waelelsafty07) [reply index.html]: Done

File: index.html (hunk 2 above)
Comment (waelelsafty07) [reply index.html:64]: done

File: styles.css (hunk 3 above)
Comment (waelelsafty07) [reply styles.css]: done

**Comment by waelelsafty07** [comment]
thanks to u @shereen-fathy

**Comment by shereen-fathy** [comment]
changes look good for me, Thank you 💯

**Pull request merged**